content/swiftui/concepts/views/terms/button/button.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'Button'
+Description: 'A very basic component in the UI, which is used to interact and perform certain actions'
+Subjects:
+  - 'Sotfware Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **button** is a very basic component in the UI, which is used to interact and perform certain actions, such as logging into a page, or deleting a file etc.
+
+## Syntax
+
+```swift
+Button("Text inside the button"){
+  //What happens when you click the button
+}
+```
+
+The `Text inside the button` is what is shown inside the button.
+
+## Example
+
+To make a button which will `count` the number of times we `click` it:
```

Comment:
```suggestion
To make a button which will count the number of times we click it:
```

we aren't referring to the code block yet so may not need these wrapped in backticks

Reply:
Okk